### PR TITLE
RegButton Removed / About & Contact Removed / Home Page Clutter Removed / Log On Button Made

### PR DESCRIPTION
**Pull request by AnMcCbusiness**
Only admins should be able to register any onboarding.

### CODE REVIEW

**Comment by AnMcCbusiness** [comment]
Removed About and Contact Links/Pages

File: ClubOOPSInventoryProject/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -1,31 +1,9 @@
...
-        <p><a class="btn btn-default" href="https://go.microsoft.com/fwlink/?LinkId=301866">Learn more &raquo;</a></p>
-    </div>
-    <div class="col-md-4">
-        <h2>Web Hosting</h2>
-        <p>You can easily find a web hosting company that offers the right mix of features and price for your applications.</p>
-        <p><a class="btn btn-default" href="https://go.microsoft.com/fwlink/?LinkId=301867">Learn more &raquo;</a></p>
-    </div>
+    <h1>Inventory Management</h1>
+    <p class="lead">A .NET web application to keep track of CPTC club inventory items.</p>
+    <p><a href="/Account/Login" class="btn btn-primary btn-lg">Log in &raquo;</a></p>
```
[the remark this answers is not in the document]
Comment (AnMcCbusiness) [reply ClubOOPSInventoryProject/Views/Home/Index.cshtml]: @thedevonthompson Sorry about that! I did a bit of digging and believe I have the correct approach with what I was going for. Commit has been added. Needed to use HttpUtility.HtmlDecode, as well to get my arrow things to work on my button.

**Comment by AnMcCbusiness** [comment]
@thedevonthompson Thanks Devon! I think I succeeded with what needs to be done, let me know if there's anything else.

**Comment by thedevonthompson** [comment]
Due to the major change to .NET Core, you'll have to create a new pull request in order to make these changes.

**Pull request closed without merging**